src/test/java/at/ac/tuwien/kr/alpha/api/mapper/impl/AnswerSetToWorkbookMapperTest.java

```diff
@@ -0,0 +1,98 @@
+package at.ac.tuwien.kr.alpha.api.mapper.impl;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.junit.Assert;
+import org.junit.Test;
+
+import at.ac.tuwien.kr.alpha.api.Alpha;
+import at.ac.tuwien.kr.alpha.common.AnswerSet;
+import at.ac.tuwien.kr.alpha.common.AnswerSetBuilder;
+import at.ac.tuwien.kr.alpha.common.Predicate;
+import at.ac.tuwien.kr.alpha.common.atoms.Atom;
+import at.ac.tuwien.kr.alpha.common.terms.Term;
+
+public class AnswerSetToWorkbookMapperTest {
+
+	private AnswerSetToWorkbookMapper mapper = new AnswerSetToWorkbookMapper();
+
+	@Test
+	public void smokeTest() throws IOException {
+		AnswerSet as = new AnswerSetBuilder().predicate("bla").instance("blubb", "blubb").instance("foo", "bar").predicate("foo").instance("bar")
+				.instance("baz").predicate("complex").instance(Integer.valueOf(1), Integer.valueOf(2), Integer.valueOf(3)).build();
+		Workbook wb = this.mapper.mapFromAnswerSet(as);
+		Assert.assertNotNull(wb.getSheet("Flags"));
+		Assert.assertNotNull(wb.getSheet("bla_2"));
+		Assert.assertNotNull(wb.getSheet("foo_1"));
+		Assert.assertNotNull(wb.getSheet("complex_3"));
+		wb.close();
+	}
+
+	@Test
+	public void solveAndWriteWorkbookTest() {
+		//@formatter:off
```

Comment:
Is this a special comment for an IDE formatter?

Reply:
yep, that's telling the Eclipse formatter not to touch code between this line and the next `//@formatter:on`. The same behavior can probably also be configured in other IDEs.